uiplib/gui/mainGui.py

```diff
@@ -92,6 +92,7 @@ def flush(self):
 
     def update_images(self):
         """Method to get images from directory."""
+        print("Updating Image List")
```

Comment:
why isnt flush called? what do you propose the usecase of the refresh button? just refresh of the UI, shouldnt this happen when flush is clicked itself?